website/src/main/resources/public/index.html

```diff
@@ -230,14 +230,14 @@ <h3>Contact the Owners</h3>
 </footer>
 
 <!-- Scripts -->
-<script src="assets/js/jquery-3.6.1.min.js" async></script>
-<script src="assets/js/jquery.scrollex.min.js" async></script>
-<script src="assets/js/jquery.scrolly.min.js" async></script>
-<script src="assets/js/browser.min.js" async></script>
-<script src="assets/js/breakpoints.min.js" async></script>
-<script src="assets/js/util.js" async></script>
-<script src="assets/js/main.js" async></script>
-<script src="https://cdn.jsdelivr.net/npm/@widgetbot/html-embed@1.0.0"></script>
+<script src="assets/js/jquery-3.6.1.min.js"></script>
+<script src="assets/js/jquery.scrollex.min.js" ></script>
+<script src="assets/js/jquery.scrolly.min.js"></script>
+<script src="assets/js/browser.min.js" ></script>
+<script src="assets/js/breakpoints.min.js" ></script>
+<script src="assets/js/util.js" ></script>
+<script src="assets/js/main.js" ></script>
+<script src="https://cdn.jsdelivr.net/npm/@widgetbot/html-embed@1.3.0"></script>
```

Comment:
while this is probably a sane choice, its likely important to figure out why `async` was added in the first place.
i would not be surprised if it was done to cut down response time for the website which in turn increases SEO massively.

so if thats the case, just undoing this might not be a good choice given that the websites primary role is to be SE-relevant.